default.yml

```diff
@@ -11,7 +11,6 @@
     os_desktop_enable: true
     os_env_extra_user_paths: ['/home']
     os_auth_allow_homeless: true
-    os_security_kernel_enable_core_dump: true
```

Comment:
Well sorry, I made a mistake here: I thought you removed the line in the `defaults/main.yml`, not in the `default.yml`. The setting in the default.yml overwrites the setting in the defaults/main.yml, so removing that line was ok. I'll fix this later though.